willow/Dockerfile

```diff
@@ -34,39 +29,19 @@ RUN apt-get update -qq && \
     ffmpeg \
     ufraw \
     bzip2 unzip xz-utils \
-    openjdk-8-jre-headless="$JAVA_DEBIAN_VERSION" \
-    ca-certificates-java="$CA_CERTIFICATES_JAVA_VERSION" \
+    # install open-jdk and ca-certs from jessie-backports
+    && apt-get install -t jessie-backports -y --no-install-recommends openjdk-8-jre-headless ca-certificates-java \
     && apt-get clean \
-	&& rm -rf /var/lib/apt/lists/*
-
-# add a simple script that can auto-detect the appropriate JAVA_HOME value
-# based on whether the JDK or only the JRE is installed
-RUN { \
-		echo '#!/bin/sh'; \
-		echo 'set -e'; \
-		echo; \
-		echo 'dirname "$(dirname "$(readlink -f "$(which javac || which java)")")"'; \
-	} > /usr/local/bin/docker-java-home && \
-	chmod +x /usr/local/bin/docker-java-home && \
-    set -x && [ "$JAVA_HOME" = "$(docker-java-home)" ]
+	&& rm -rf /var/lib/apt/lists/* \
+	&& /var/lib/dpkg/info/ca-certificates-java.postinst configure \
+    && mkdir -p /fits/ \
```

Comment:
I know we'd like to reduce the number of Docker commands used, but I think we can spare a separate RUN command for installing fits, as opposed to putting it at the end of the apt-get install process.